.doc_gen/metadata/redshift_metadata.yaml

```diff
@@ -320,10 +320,8 @@ redshift_CreateTable:
   services:
     redshift: {CreateTable}
 redshift_Scenario:
-  title: Get started with &RS; tables, items, and queries
-  title_abbrev: Get started with &RS;
   synopsis: work with &RS; tables, items, and queries.
-  category: Scenarios
+  category: Basics
```

Comment:
I'd like to see a synopsis_list for this example also.

Reply:
Scope creep.